hack/local-up-cluster.sh

```diff
@@ -247,17 +286,21 @@ function start_controller_manager {
 }
 
 function start_kubelet {
+    echo "Start kubelet"
     KUBELET_LOG=/tmp/kubelet.log
+ 
+    # The kubelet in this example access the API Server through a service endpoint,
+    # in order for this to happen, we need to provide kubeconfig credentials as well.
     if [[ -z "${DOCKERIZE_KUBELET}" ]]; then
-      sudo -E "${GO_OUT}/kubelet" ${priv_arg}\
+      sudo -E "${GO_OUT}/kubelet" ${priv_arg} \
         --v=${LOG_LEVEL} \
         --chaos-chance="${CHAOS_CHANCE}" \
         --container-runtime="${CONTAINER_RUNTIME}" \
         --rkt-path="${RKT_PATH}" \
-        --rkt-stage1-image="${RKT_STAGE1_IMAGE}" \
         --hostname-override="127.0.0.1" \
-        --address="127.0.0.1" \
-        --api-servers="${API_HOST}:${API_PORT}" \
+        --rkt-stage1-image="${RKT_STAGE1_IMAGE}" \
+        --api-servers="https://127.0.0.1:30279" \
```

Comment:
I'd almost prefer a random port to yet-another-parameter that nobody ever edits.... But I guess I can set it to a arg w/ default value.